recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "swifter" %}
```

Comment:
i believe this is the only file i updated - i assume the others were auto-updated by conda-forge?

Reply:
Yes, they got updated when you rerender and should never be changed manually.